projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -26,16 +26,17 @@ def run_job(path, arguments=None):
                 return "Incorrect path!"
         with open("vdk_logs.txt", "w+") as log_file:
             path = shlex.quote(path)
-            arguments = shlex.quote(arguments)
-            cmd_str = (
-                f"vdk run {path}"
-                if not arguments
-                else f"vdk run {path} --arguments {arguments}"
-            )
-            cmd = shlex.split(cmd_str)
+            cmd: list[str] = ["vdk", "run", f"{path}"]
+            if arguments:
+                arguments = shlex.quote(arguments)
+                cmd.append("--arguments")
+                cmd.append(f"{arguments}")
             process = subprocess.Popen(
-                cmd, stdout=log_file, stderr=log_file, env=os.environ.copy()
+                cmd,
+                stdout=log_file,
+                stderr=log_file,
+                env=os.environ.copy(),
+                shell=False,
             )
-            subprocess.check_output(["/bin/ls", "-l"])
```

Comment:
why is the line cleared?

Reply:
This was there for testing purposes, I have missed to delete it in my previous commit